configs/_base_/schedules/schedule_adam_step_12e.py

```diff
@@ -0,0 +1,11 @@
+# optimizer
+optimizer = dict(type='Adam', lr=4e-4)
+optimizer_config = dict(grad_clip=None)
+# learning policy
+lr_config = dict(
+    policy='step',
+    warmup='linear',
+    warmup_iters=100,
+    warmup_ratio=1.0 / 3,
+    step=[11])
+runner = dict(type='EpochBasedRunner', max_epochs=12)
```

Comment:
`chekpoint_config` has been removed from the `default_runtime.py` in the latest repo. Therefore, it should add `checkpoint_config = dict(interval=1)` in the schedule config here.